add new overview topic (#2821)
influxdata/docs-v2: content/influxdb/cloud/notebooks/overview.md

---
title: Overview of notebooks
description: >
  Learn about the building blocks of a notebook. 
weight: 101
influxdb/cloud/tags:
menu:
  influxdb_cloud:
    name: Overview of notebooks
    parent: Notebooks
---

> 

Reviewer: @mluu12 alternatively, we could pull some content from the main Notebooks page over to this doc, and expand the intro a bit

  - Choose a bucket to define your data source.
  - Edit your time range with the time range option in the dropdown menu.
  - Add filters to narrow your data.
- **Transform: Flux Script**:
  - Use `__PREVIOUS_RESULT__` to build from data in the previous cell.
  - Enter a Flux script to transform your data.
- **Transform: Downsample**:
  - Window data by time and apply an aggregate to each window to downsample data.
  - For more information, see [Downsample data with notebooks](/influxdb/cloud/notebooks/downsample/).
- **Pass-through: Markdown**: Enter explanatory notes or other information for yourself or one of your team members in Markdown.
- **Pass-through: Visualization**:
  - Create a visualization of your data. For details on available visualization types and how to use them, see [Visualization types](/influxdb/cloud/visualize-data/visualization-types/).
  - Click **Export to Dashboard** in the upper right to send your cell to a new or existing dashboard.
- **Output: Output to Bucket**: Write data to a bucket. Click **Run** in the upper left to write a single time, or select **Export as Task** to schedule your output as a task.

## Notebook controls

> The following options appear in the upper right of each notebook.

Reviewer: ```suggestion
The following controls appear at the top of each notebook:
- [Preview/Run option](#previewrun-mode)
- [Presentation mode](#presentation-mode)
- [Local/UTC timezone](#local-or-utc-timezone)
- [Time range](#time-range)
```

> 

Reviewer: ```suggestion

### Preview/Run mode
- Select **Preview** to display results of each cell without writing data. Helps to verify that cells return expected results before writing data.
- Select **Run** mode displays results of each cell and writes data to the selected bucket.
```

### Presentation mode
Toggle the **Presentation** option to enter presentation mode. Use presentation mode to view notebooks in full screen, hiding the left and top navigation menus so only the cells appear. This mode might be helpful, for example, for stationary screens dedicated to monitoring visualizations.

### Time range
Select from the options in the dropdown list or select **Custom Time Range** to enter a custom time range with precision up to nanoseconds.

### Timezone

> Click the timezone dropdown list to select a timezone to use for the dashboard. Select either the local time (default) or UTC.

Reviewer: ```suggestion
### Local or UTC timezone
Click the timezone dropdown list to select a timezone to use for the dashboard. Select either the local time (default) or UTC.

### Time range
Select from the options in the dropdown list or select **Custom Time Range** to enter a custom time range with precision up to nanoseconds.
```
